client/blocks/signup-form/passwordless.jsx

```diff
@@ -75,6 +71,16 @@ class PasswordlessSignupForm extends Component {
 			password: '',
 		};
 
+		// If not in a flow, submit the form as a standard signup form.
+		// Since it is a passwordless form, we don't need to submit a password.
+		if ( this.props.flowName === '' ) {
+			this.props.submitForm( { ...form, is_passwordless: true } );
```

Comment:
What if we don't have a `flowName` and no `submitForm` property?

Reply:
Let's also check if these properties sent from `passowrdless`( a little down here ) are also sent here:

```
		anon_id: getTracksAnonymousUserId(),
		is_dev_account: ref === 'developer-lp',
```

Reply:
We can add `is_dev_account` for sure 👍 Just not sure about `anon_id`, seems that it is not an API fbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf.cuc%3Se%3Q49pn245r%234877-og

Reply:
What do you think of adding this message when `flowName` and `submitForm` are empty? This is very unlike to happen, but nice to handle.


<img width="410" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/3b172341-9e1e-40f1-a5c4-5041f49c0617">

Reply:
👍 make sense